pkg/server/commands/list_objects.go

```diff
@@ -259,6 +261,10 @@ func (q *ListObjectsQuery) Execute(
 
 		case result, channelOpen := <-resultsChan:
 			if result.Err != nil {
+				code := status.Code(result.Err)
```

Comment:
We should be using our errors package here instead of the grpc codes directly.

`serverErrors.HandleError("", err)`

Reply:
Updated to use serverErrors.HandleError is possible but treat AuthorizationModelResolutionTooComplex separately